ChildProject/metrics.py

```diff
@@ -244,7 +244,7 @@ def gamma(
         )
 
     dissim = CombinedCategoricalDissimilarity(
-        list(continuum.categories), delta_empty=1, alpha=alpha, beta=beta
+        delta_empty=1, alpha=alpha, beta=beta
```

Comment:
this is unrelated, right?

Reply:
yes; the tests were failing due to an upgrade of pygamma-agreement. had to change this for the tests to pass again